config.js

```diff
@@ -112,11 +48,14 @@ exports.updateFromArgv = function() {
         .describe('noclock', 'Do not send any clock/time data to the bot')
         .describe('persist', 'Bot process remains running between moves')
         .describe('corrqueue', 'Process correspondence games one at a time')
-        .describe('maxtotalgames', 'Maximum number of total games')
-        // maxtotalgames is actually the maximum total number of connected games for your bot 
-        // which means the maximum number of games your bot can play at the same time (choose a low number to regulate your GPU use)
+        .describe('maxconnectedgames', 'Maximum number of connected games for all users')
+        .default('maxconnectedgames', 20)
+        // maxconnectedgames is actually the maximum number of connected games for all users 
+        // against your bot, which means the maximum number of games your bot can play at the same time 
+        // (choose a low number to regulate your computer performance and stability)
         // (correspondence games are currently included in the total connected games count if you use `--persist` )
```

Comment:
I don't believe --persist matters, right? Connected game list is every game you are a part of on OGS. Maybe change to:

> (correspondence games are currently included in the total connected games count)

Reply:
indeed yes
weird i'm sure i had removed this line some time ago